docs/guides/latest/volumesnapshot/pvc.md

```diff
@@ -0,0 +1,441 @@
+  name: source-pod
+  namespace: demo
+spec:
+  containers:
+    - name: busybox
+      image: busybox
+      command: ["/bin/sh", "-c"]
+      args: ["touch /demo/data/sample-file.txt && sleep 3000"]
+      volumeMounts:
+        - name: source-data
+          mountPath: /source/data
+  volumes:
+    - name: source-data
+      persistentVolumeClaim:
+        claimName: source-pvc
+        readOnly: false
+```
+
+Let's create the Pod we have shown above.
+
+```console
+$ kubectl apply -f ./docs/examples/guides/latest/volumesnapshot/standalone-pvc/source-pod.yaml
+pod/source-pod created
+```
+
+Now, wait for Pod to go into the Running state.
+
+```console
+$ kubectl get pod -n demo 
+NAME         READY   STATUS    RESTARTS   AGE
+source-pod   1/1     Running   0          25s
+```
+
+Verify that the sample data has been created in /source/data directory for `source-pod`
+using the following command,
+
+```console
+$ kubectl exec -n demo source-pod  -- ls -R /demo/data
+/demo/data:
+lost+found
+sample-file.txt
+```
+
+**Create BackupConfiguration :**
+
+Now, create a `BackupConfiguration` crd to take snapshot of the `source-pvc` PVC.
+
+Below is the YAML of the `BackupConfiguration` crd that we are going to create,
+
+```yaml
+apiVersion: stash.appscode.com/v1beta1
+kind: BackupConfiguration
+metadata:
+  name: pvc-volume-snapshot
+  namespace: demo
+spec:
+  schedule: "*/1 * * * *"
+  driver: VolumeSnapshotter
+  target:
+    ref:
+      apiVersion: v1
+      kind: PersistentVolumeClaim
+      name: source-pvc
+    snapshotClassName: default-snapshot-class
+  retentionPolicy:
+    name: 'keep-last-5'
+    keepLast: 5
+    prune: true
+```
+
+Here,
+
+- `spec.schedule` is a [cron expression](https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#schedule) indicates that `BackupSession` will be created at 1 minute interval.
+
+- `spec.driver` indicates the name of the agent to use to back up the target. Currently, Stash supports `Restic`, `VolumeSnapshotter` drivers. The `VolumeSnapshotter` is used to backup/restore PVC using `VolumeSnapshot` API.
+
+- `spec.target.ref`  refers to the backup target. `apiVersion`, `kind` and `name` refers to the `apiVersion`, `kind` and `name` of the targeted workload respectively. Stash will use this information to create a Volume Snapshotter Job for creating VolumeSnapshot.
+
+- `spec.target.snapshotClassName` indicates the [VolumeSnapshotClass](https://kubernetes.io/docs/concepts/storage/volume-snapshot-classes/) to be used for volume snapshotting.
+
+Let's create the `BackupConfiguration` crd we have shown above.
+
+```console
+$ kubectl apply -f ./docs/examples/guides/latest/volumesnapshot/standalone-pvc/backupconfiguration.yaml
+backupconfiguration.stash.appscode.com/pvc-volume-snapshot created
+```
+
+**Verify CronJob :**
+
+If everything goes well, Stash will create a `CronJob` to take periodic snapshot of the PVC with the schedule specified in `spec.schedule` field of `BackupConfiguration` crd.
```

Comment:
Does this work with already mounted PVC or PVC has to be bound to a PV but not mounted into any pod?

Reply:
CSI driver can take backup of a PVC while it is mounted in a pod.